src/QUICConnection.ts

```diff
@@ -990,8 +1006,16 @@ class QUICConnection {
       if (quicStream == null) {
         if (this[running] === false || this[status] === 'stopping') {
           // We should reject new connections when stopping
-          this.conn.streamShutdown(streamId, Shutdown.Write, 1);
-          this.conn.streamShutdown(streamId, Shutdown.Read, 1);
+          this.conn.streamShutdown(
+            streamId,
+            Shutdown.Write,
+            this.reasonToCode('write', errors.ErrorQUICConnectionStopping),
+          );
```

Comment:
Does the end user ever see the "code" for this?

Reply:
I believe so, yeah. Streams get codes when shutdown. Connections get codes and a message.